_data/simple-icons.json

```
@@ -617,6 +617,17 @@
             "source": "https://aws.amazon.com/architecture/icons/",
             "guidelines": "https://aws.amazon.com/architecture/icons/"
         },
+        {
+            "title": "Amazon Organizations",
+            "hex": "E7157B",
+            "source": "https://aws.amazon.com/architecture/icons/",
+            "guidelines": "https://aws.amazon.com/architecture/icons/",
+            "aliases": {
+                "aka": [
+                    "AWS Organizations"
+                ]
+            }
```

```suggestion
            "title": "AWS Organizations",
            "hex": "E7157B",
            "source": "https://aws.amazon.com/architecture/icons",
            "guidelines": "https://aws.amazon.com/architecture/icons"
```

It's referred to internally as AWS Organisations, and in keeping with our other AWS/Amazon icons to date, should be formatted like this.